argostime/crawler/intergamma.py

```diff
@@ -40,6 +40,9 @@ def crawl_intergamma(url: str) -> CrawlResult:
         logging.error("Got status code %s while getting url %s", response.status_code, url)
         raise PageNotFoundException(url)
 
+    # Use UTF-8 encoding instead of ISO-8859-1
+    response.encoding = 'UTF-8'
```

Comment:
Is this something we might need to do for all crawlers or just this one?

Reply:
I don't think so. I checked the encoding for Gamma and Karwei and it said `ISO-8859` but if you check the source of the page you can find this meta tag in the head: `<meta charset="utf-8"/>`.

So, I think the http header and HTML head simply do not agree on Intergamma sites.